### ISSUE TITLE
Can't using compute_feature in feature metric (IS score)

### ISSUE THREAD

**Issue by quyet0nguyen** [photosynthesis-team/piq#249]
**Describe the bug**
When I tries using compute_feature in feature metric. I always have an error:
![image](https://user-images.githubusercontent.com/43262313/114641998-4b5d5980-9cfd-11eb-9058-518b5332371a.png)

my code is here:
![image](https://user-images.githubusercontent.com/43262313/114642032-5adca280-9cfd-11eb-98a2-3bb79b8f5dd2.png)

I has used several image metrics, It so good.Thanks you so much!

**Comment by zakajd** [comment]
Hi @quyet0nguyen 
As written in the docstring for `compute_feats` method, we expect dataloader to return a _dictionary_ with key `images` in it.
Standart DataLoader returns tuples of `(image, label)`, so here is quick hack to make it work:
```python
class DictDataLoader(DataLoader):
    def __getitem__(self, idx):
        batch = super().__getitem__(self, idx)
        return {'images': batch[0], 'labels': batch[1]}
first_dl, second_dl = DictDataLoader('/content/test'), DictDataLoader('/content/generate')
```

**Comment by quyet0nguyen** [comment]
@zakajd  thanks you very much
